src/pybind/mgr/dashboard/requirements.txt

```diff
@@ -9,3 +9,4 @@ Routes
 prettytable
 pytest
 pyyaml
+natsort
```

Comment:
needed?

Reply:
this one is needed, otherwise UT of dashboard will fail.